armando/block-template-parts/footer.html

```diff
@@ -1,49 +1,48 @@
-<!-- wp:columns {"align":"wide"} -->
-<div class="wp-block-columns alignwide">
-	<!-- wp:column -->
-	<div class="wp-block-column">
-		<!-- wp:latest-posts /-->
-	</div>
-	<!-- /wp:column -->
-
-	<!-- wp:column -->
-	<div class="wp-block-column">
-		<!-- wp:paragraph {"align":"center"} -->
-		<p class="has-text-align-center"><strong>Never miss the latest news</strong></p>
-		<!-- /wp:paragraph -->
-
-		<!-- wp:buttons {"contentJustification":"center"} -->
-		<div class="wp-block-buttons is-content-justification-center">
-			<!-- wp:button {"borderRadius":3,"backgroundColor":"light-blue","textColor":"dark-blue"} -->
-			<div class="wp-block-button"><a
-					class="wp-block-button__link has-dark-blue-color has-light-blue-background-color has-text-color has-background"
-					style="border-radius:3px">Join our mailing list</a></div>
-			<!-- /wp:button -->
+<!-- wp:group {"align":"full","style":{"color":{"link":"var:preset|color|white"}},"backgroundColor":"dark-blue","textColor":"white"} -->
```

Comment:
Because styling of template parts is not reflected across the site, I had to move the colors to the group.